troubleshooting/extra-android-permissions-always-set.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Extra Android permissions always set
+description: The APK built by the cloud service has more required permissions than I set in my project settings.
+type: troubleshooting
+page_title: Unwanted permissions on Android build
+slug: extra-android-permissions-always-set
+position:
+tags: appbuilder, android-permissions
+teampulseid:
+ticketid: 838093, 1082473
+pitsid:
+
+---
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>{{site.ab-s}} ({{site.ab}})</td>
+  <tr>
+    <td>Mobile development type</td>
+    <td>{{site.ac}} app</td>
+  </tr>
+  <tr>
+    <td>Mobile OS</td>
+    <td>Android</td>
+  </tr>
+</table>
+
+## Description
+When I build my application the app asks for permissions that I didn’t set up. Permissions are added although not included in any way in the `config.xml` or the `AndroidManifest.xml`.
+
+## Cause\Possible Cause(s)
+Some core Cordova plugins require certain Android permissions and they add these permissions to the `AndroidManifest.xml` at build time.
+
+Using plugins is essentials in Cordova applications to access device features which on other hand require setting Android permissions.
+
+Each plugin injects its required permissions in the `AndroidManifest.xml` file during build time - this is both done for convenience (you do not need to manually enter each required permission by the used plugins) but also to make sure that all required permissions are not omitted or deliberately removed as this would cause the plugin that requires them to not work properly.
+
+## Solution
+1. In order to reduce the number of permissions, disable any plugins that you are not using.
+For example, if the app does not read, delete or add contacts, you should disable the core Contacts plugin. You can do that by following these steps:
+    - Open the app.
+    - From the **Solution Explorer** select **Properties**.
+    - Navigate to the **Plugins** tab.
+    - Locate the **Contacts** plugin.
+    - On the left side, click on the **switch** to disable the plugin.

```

Comment:
Somehow I do not believe that this KB should contain information on how to remove plugins. Perhaps you can add a link to the docs with info on how to manage [Core plugins](http://docs.telerik.com/platform/appbuilder/cordova/using-plugins/using-core-plugins/enable-disable-core) and remove this part.